File: py/ege9.99.py
f = open("ege9.99.txt")

smax = 0

for l in f:
    if l == '\n': continue
    ax = list(map(int, l.split())) 
    
    m = max(ax)
    if sum(x * x for x in ax) == 2 * m**2:
        s = sum(ax) - m
        if s > smax:
            smax = s

print(smax)

File: c/ege9.99.c
#include <stdio.h>
#include "array.lib.c"

int main() {
    FILE *f = fopen("ege9.99.txt", "r");
    int ax[3];
    int m;
    int smax = 0;
    int s;

    while(freadn(f, ax, 3) == 3) {
        m = max(ax, 3);
        if(sumsqr(ax, 3) == 2 * m * m) {
            s = sum(ax, 3) - m;
            if(s > smax) {
                smax = s;
            }
        }
    }

    printf("%d\n", smax);
}
